frontend/src/css/Sidebar.styl

```diff
@@ -55,8 +60,10 @@ borderWhenActive= 5px
       border-inline-start-style solid
       border-inline-start-width borderWhenActive
       font-weight bold
+      &.sidebar__item__inside_menu
+        padding-inline-start spacingInsideMenu - 5px
       &.sidebar__item__space
-        padding-inline-start 0
+        padding-inline-start spacingInsideMenu + 5px
```

Comment:
Might be _nothing_ but shouldn't we have a defined variable for ``space`` padding?

Reply:
Fixed